webpack/scenes/Subscriptions/Manifest/__tests__/manifest.fixtures.js

```diff
@@ -210,7 +249,20 @@ export const deleteManifestFailureActions = [
     type: 'DELETE_MANIFEST_REQUEST',
   },
   {
-    result: new Error('Request failed with status code 422'),
     type: 'DELETE_MANIFEST_FAILURE',
+    payload: {
+      message: 'Request failed with status code 422',
+      result: new Error('Request failed with status code 422'),
+    },
+  },
+  {
+    payload: {
+      message: {
+        message: 'Request failed with status code 422',
+        sticky: true,
+        type: 'error',
+      },
+    },
+    type: 'TOASTS_ADD',
```

Comment:
I know this is just a fixture but do we want to combine all of these into a `const` so we don't have to repeat ourselves here?